### ISSUE TITLE
Update dependencies?

### ISSUE THREAD

**Issue by jaredbeck** [beauby/jsonapi#54]
jsonapi-parser and jsonapi-renderer have new versions, currently 0.1.1 and 0.1.2, respectively, but the latest release of jsonapi, 0.1.1.beta6, depends on 0.1.1.beta3 and 0.1.1.beta1, respectively.

Should we release a new version of jsonapi with updated dependency constraints, or should people (e.g. active_model_serializers) no longer use the jsonapi gem? I mean, should people just use jsonapi-parser and jsonapi-renderer directly?

**Comment by beauby** [comment]
You are right. The `jsonapi` gem was previously just a bundle of `jsonapi-serializer` and `jsonapi-renderer`, and AMS is using only a helper class of `jsonapi-renderer` (namely `JSONAPI::IncludeDirective`). The AMS dependency was previously not properly pinned to a specific version, which I saw as a risk for many users, so I avoided updating this gem. Moreover, the name `jsonapi` being somewhat too generic for what this gem evolved into (namely [jsonapi-rb](http://jsonapi-rb.org), which bundles `jsonapi-renderer` and `jsonapi-parser`, along with serializers and deserializers, with tight integrations with various frameworks), I decided to stay away from it for fairness.

TL;DR: Yes, people should use `jsonapi-parser` and `jsonapi-renderer` directly (or give a try to [jsonapi-rb](http://jsonapi-rb.org), depending on their needs).

**Comment by jaredbeck** [comment]
Thanks Lucas, I've opened a PR asking AMS to just use `jsonapi-renderer` (https://github.com/rails-api/active_model_serializers/pull/2055)
